doc/source/data/transforming-data.rst

```diff
@@ -115,19 +100,87 @@ uses tasks by default.
         .map_batches(increase_brightness)
     )
 
-.. _transforming_data_actors:
+.. _configure_batch_format:
```

Comment:
Github somehow shows the change, but this section (`Configuring batch format`) is not changed.